pkg/lib/cockpit.d.ts

```diff
@@ -368,4 +368,18 @@ declare module 'cockpit' {
 
     export let localStorage: Storage;
     export let sessionStorage: Storage;
+
+    export interface HttpOptions {
+        address: string;
+        port: number;
+        headers: { [key: string]: string };
+      }
```

Comment:
All of these are optional. It's missing the "tls" and "superuser" properties, see https://cockpit-project.org/guide/latest/cockpit-http.html#cockpit-http-constructor, plus the standard `binary` flag